java/cdk/src/main/java/sleeper/cdk/stack/QueryStack.java

```diff
@@ -162,16 +164,16 @@ private IFunction createFunction(String id, LambdaCode queryJar, InstancePropert
     }
 
     /***
-     * Creates the queue and Lambda needed for QueriesQuery
+     * Creates the Lambda needed for QueriesQuery
```

Comment:
There's a lot of "QueriesQuery" in this file. Is it worth tidying up these names now?

Reply:
I've raised this issue for tidying up the names: https://github.com/gchq/sleeper/issues/1795